managed/services/management/dbaas/psmdb_cluster_service.go

```diff
@@ -124,7 +124,22 @@ func (s PSMDBClusterService) CreatePSMDBCluster(ctx context.Context, req *dbaasv
 		return nil, err
 	}
 
-	if err := s.fillDefaults(ctx, kubernetesCluster, req); err != nil {
+	psmdbComponents, err := s.componentsService.GetPSMDBComponents(ctx, &dbaasv1beta1.GetPSMDBComponentsRequest{
+		KubernetesClusterName: kubernetesCluster.KubernetesClusterName,
+	})
+	if err != nil {
+		return nil, errors.New("cannot get the list of PXC components")
+	}
+
+	backupImage := ""
+	backupComponent, err := DefaultComponent(psmdbComponents.Versions[0].Matrix.Backup)
```

Comment:
Is it possible that `psmdbComponents.Versions` will be empty? If yes we will have panic here.